packages/web/app/forms/ProcedureForm/ProcedureAdditionalData.jsx

```diff
@@ -66,6 +67,7 @@ export const ProcedureAdditionalData = ({
 }) => {
   const api = useApi();
   const { currentUser, facilityId } = useAuth();
+  const [unSavedChangesModalOpen, setUnSavedChangesModalOpen] = useState(false);
```

Comment:
```suggestion
  const [unsavedChangesModalOpen, setUnSavedChangesModalOpen] = useState(false);
```
In my head this is one word "unsaved"...nit though, no biggy

Reply:
And actually in this file, it doesn't test for dirtiness...you've given the component a name that reflects that (`CancelAdditionalDataModal`) and I reckon the state might as well have the same